TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/ProxyClass.php

```diff
@@ -43,32 +45,32 @@ class ProxyClass
     protected $fullOriginalClassName;
 
     /**
-     * @var \TYPO3\Flow\Object\Proxy\ProxyConstructor
+     * @var ProxyConstructor
      */
     protected $constructor;
 
     /**
      * @var array
      */
-    protected $methods = array();
+    protected $methods = [];
 
     /**
      * @var array
      */
-    protected $constants = array();
+    protected $constants = [];
 
     /**
      * @var array
      */
-    protected $interfaces = array('\TYPO3\Flow\Object\Proxy\ProxyInterface');
+    protected $interfaces = ['\TYPO3\Flow\Object\Proxy\ProxyInterface'];
```

Comment:
I tried that but a ::class doesn't has a leading "\" and a `protected $interfaces = ['\' . ProxyInterface::class];` is not valid in PHP 5.6